src/index.html

```diff
@@ -11,7 +11,168 @@
+            pattern="\d+"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label for="dob" class="form_field">
+            Full date of birth:
+          <input
+            type="date"
+            name="dob"
+            id="dob"
+          >
+        </label>
+
+        <label for="check" class="form_field">
+            I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            id="check"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label for="email" class="form_field">
+          E-mail:
+          <input type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label for="password" class="form_field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            > Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="no"
+            >
+              No
+          </label>
+        </label>
+        <label for="color" class="form_field">
+            What is your favorite color?
+          <input
+            type="color"
+            name="password"
+            id="color"
+          >
+        </label>
+
+        <label for="time" class="form_field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            id="time"
+          >
+        </label>
+
+        <label for="car" class="form_field">
+          What are your favorite brand of cars?
+          <select
+            name="car"
+            id="car"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label for="rate" class="form_field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            id="rate"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label for="comments" class="form_field">
+          Comments:
+          <textarea
+              name="comments"
+              id="comments"
+              autocomplete="off">
```

Comment:
```suggestion
          <textarea
            name="comments"
            d="comments"
            autocomplete="off"
          >
```

Reply:
thank you, changed these lines in a new commit